README.md

```diff
@@ -1,2 +1,5 @@
-# trials
-Review microverse trials project
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
You will agree with me that you did describe your project in the readme file.
A description of projects helps others under what your project is all about and let others understand how to use your project

Reply:
- kindle makes sure you make your comment more polite, to do that start your comment with `please, kindly` to recognize the effort the student has put to make the project work and to make the student not feel offended with your request changes :grinning: :grinning: :grinning: